ios/FluentUI.Demo/FluentUIDemoTests/CommandBarTest.swift

```diff
@@ -12,4 +12,79 @@ class CommandBarTest: BaseTest {
     func testLaunch() throws {
         XCTAssertTrue(app.navigationBars[controlName].exists)
     }
+
+    // ensures that adding (refreshing)/removing leading/trailing button works as expected
+    func testLeadingTrailingButton() throws {
+        XCTAssert(app.otherElements["Command Bar with 1 leading button and 0 trailing buttons"].exists)
+        app.buttons["Refresh Trailing Button"].tap()
+        XCTAssert(app.otherElements["Command Bar with 1 leading button and 1 trailing button"].exists)
+        app.buttons["Remove Leading Button"].tap()
+        XCTAssert(app.otherElements["Command Bar with 0 leading buttons and 1 trailing button"].exists)
+        app.buttons["Remove Trailing Button"].tap()
+        XCTAssert(app.otherElements["Command Bar with 0 leading buttons and 0 trailing buttons"].exists)
+        app.buttons["Refresh Leading Button"].tap()
+        XCTAssert(app.otherElements["Command Bar with 1 leading button and 0 trailing buttons"].exists)
+    }
+
+    // ensures that tapping on text style button rotates through text styles
+    func testChangeTextStyleButton() throws {
+        let textStyleButton: XCUIElement = app.buttons.element(boundBy: 15)
```

Comment:
Can we move the magic numbers in `element(boundBy: <NUM>)` across this file to actual constants?